docs/docs/action-server/add-sanic-extensions.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: add-sanic-extensions
+sidebar_label: Adding Sanic Extensions
+title: Add Sanic Extensions
+---
+
+# Add Sanic extensions
+With the use of Pluggy, you can now create additional sanic extensions by accessing the app object created by the action server. The hook implemented provides you access 
+to sanic app object created by the rasa sdk when starting the action server. Using this app object, you can now extend Sanic features such as middlewares, listeners, background tasks
```

Comment:
```suggestion
to the Sanic app object created by `rasa-sdk` when starting the action server. Using this app object, you can now extend Sanic features such as middlewares, listeners, background tasks
```

Reply:
This should be addressed too.

Reply:
i think my rebase overwrote all the changes i did here :( i will redo it.